dask_ml/linear_model/glm.py

```diff
@@ -245,7 +249,11 @@ def predict_proba(self, X):
             The probability of the sample for each class in the model.
         """
         X_ = self._check_array(X)
-        return sigmoid(dot(X_, self._coef))
+        prob = sigmoid(dot(X_,self._coef)) # TODO: verify, multi_class broken
+        if self.n_classes==2:
+            return np.vstack([1 - prob, prob]).T
```

Comment:
This needs to be able to handle dask arrays too. You can look at how functions like `sigmoid` are dispatched.

Reply:
I will look into it @TomAugspurger